Lib/test/support/interpreters/channels.py

```diff
@@ -6,8 +6,7 @@
 
 # aliases:
 from _interpchannels import (
-    ChannelError, ChannelNotFoundError, ChannelClosedError,
-    ChannelEmptyError, ChannelNotEmptyError,
```

Comment:
This might be an intentional re-export? cc @ericsnowcurrently as `test.support.interpreters` follows the layout of the (hopefully) soon-to-be `interpreters` module

Reply:
Oh right, these imports should be kept. Add `# noqa: F401`.